hydra-cardano-api/src/Hydra/Cardano/Api/Point.hs

```diff
@@ -14,3 +17,18 @@ toConsensusPointHF ::
 toConsensusPointHF ChainPointAtGenesis = Consensus.GenesisPoint
 toConsensusPointHF (ChainPoint slotNo (serialiseToRawBytes -> headerHash)) =
   Consensus.BlockPoint slotNo (Consensus.OneEraHash (toShort headerHash))
+
+fromConsensusPointHF ::
+  Consensus.HeaderHash block ~ Consensus.OneEraHash xs =>
+  Consensus.Point block ->
+  ChainPoint
+fromConsensusPointHF Consensus.GenesisPoint = ChainPointAtGenesis
+fromConsensusPointHF (Consensus.BlockPoint slot (Consensus.OneEraHash h)) =
+  ChainPoint slot (fromJust $ deserialiseFromRawBytes (proxyToAsType Proxy) (fromShort h))
```

Comment:
Was a quick and dirty hack, need fix indeed!

Reply:
The reason behind this one is because cardano-api does not expose the necessary constructors to construct hashes. So, we could try to make a PR upstream to either expose the constructors or give smart constructors. In the meantime though, there isn't much way around it.